Coop/styles/Vocab/CoopAbbreviations/accept.txt

```diff
@@ -1,6 +1,7 @@
 AAD
-Ajax
+[aA]jax
 ABAP
+[aA]bap
```

Comment:
Why do we need the alternate spelling?

Reply:
As far as I have been able to figure out APAB is written with capital letters.

Reply:
We do both.

Reply:
Ok